807-registries-known-implementations.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Known implementations of CNAB Registry
+weight: 807
+---
+
+TODO
```

Comment:
We have this information at https://cnab.io/registries/. Let's at least link to it if we can't fill in more.